README.md

```diff
@@ -55,6 +55,11 @@ To create a new keypair and certificate for a certname:
 puppetserver ca generate --certname foo.example.com
 ```
 
+For enabling verbose mode:
```

Comment:
Just to be consistent with the wording we use for the other commands, I would suggest:
```suggestion
To enable verbose mode:
```